mods/stairs/init.lua

```diff
@@ -593,3 +593,22 @@ stairs.register_stair_and_slab(
 	"Gold Block Slab",
 	default.node_sound_metal_defaults()
 )
+stairs.register_stair_and_slab(
+	"ice",
+	"default:ice",
+	{cracky = 3, puts_out_fire = 1},
```

Comment:
Maybe this should have `cools_lava = 1` too?

[see here](https://github.com/minetest/minetest_game/blob/master/mods/default/nodes.lua#L557)